src/inputs/ObjectSearchCreate.tsx

```diff
@@ -42,6 +42,7 @@ export interface IObjectSearchCreateProps {
   fieldDecorator: <T>(component: T) => T;
   formManager: FormManager;
   formModel: IModel;
+  isOptionDisabled?: (option: any) => boolean;
```

Comment:
Replace the `any` with `IEndpointOption`? That's what the object search props indicates